requirements-test.txt

```diff
@@ -1,3 +1,4 @@
+PyMySQL
 pytest
 pytest-cov
 -e .
```

Comment:
Use `-e .[MySQL]` rather than adding PyMySQL to this file, so that it gets the MySQL deps from the package configuration instead.